frontend/src/metabase/visualizations/components/ObjectDetail/ObjectRelationships.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import IconBorder from "metabase/components/IconBorder";
+import LoadingSpinner from "metabase/components/LoadingSpinner";
+import { foreignKeyCountsByOriginTable } from "metabase/lib/schema_metadata";
+import { inflect } from "inflection";
+import Icon from "metabase/components/Icon";
+import { t } from "ttag";
+import cx from "classnames";
+import { ForeignKey } from "metabase-types/types/ForeignKey";
```

Comment:
Please sort imports according to our [style guide](https://www.metabase.com/docs/latest/developers-guide/frontend.html#javascript-conventions)